engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -171,6 +171,8 @@ private void initMainDirectionalLight() {
         mainDirectionalLight.lightType = LightComponent.LightType.DIRECTIONAL;
         mainDirectionalLight.lightAmbientIntensity = 0.75f;
         mainDirectionalLight.lightDiffuseIntensity = 0.75f;
+        mainDirectionalLight.lightSpecularIntensity = 0.02f;
```

Comment:
Make sunlight much less shiny. (see previous screenshots with what 0.1 was like for sunlight)

![terasology-160413203924-1280x720](https://cloud.githubusercontent.com/assets/5922109/14516535/d1f5f28a-
01b8-11e6-8fac-e7e34275914e.jpg)